spec/compiler/CodeSectionSpec.js

```diff
@@ -57,6 +57,16 @@ describe("CodeSection", function() {
       section.appendDoc("world -->");
       expect(section.doc).toEqual("hello\nworld\n");
     });
+    it("does not trim code bloks", function() {
```

Comment:
there is already one above:
it("trims content", function() {
      section.appendDoc("<!-- hello");
      section.appendDoc("world -->");
      expect(section.doc).toEqual("hello\nworld\n");
    });